saleor/graphql/plugins/types.py

```diff
@@ -2,14 +2,15 @@
 
 import graphene
 
-from ...extensions import ConfigurationTypeField, manager, models
+from ...plugins import manager, models
+from ...plugins.base_plugin import ConfigurationTypeField
 from ..core.connection import CountableDjangoObjectType
 from .enums import ConfigurationTypeFieldEnum
 
 if TYPE_CHECKING:
     # flake8: noqa
     from django.contrib.postgres.fields import JSONField
-    from ...extensions.base_plugin import PluginConfigurationType
+    from ...plugins.base_plugin import PluginConfigurationType
```

Comment:
You already imported from this file ☝️

Reply:
This import is related with type checking.